Lib/test/test_msvcrt.py

```diff
@@ -115,6 +115,28 @@ def test_heap_min(self):
         except OSError:
             pass
 
+    def test_GetErrorMode(self):
+        msvcrt.GetErrorMode()
+
+    def test_SetErrorMode(self):
+        old = msvcrt.SetErrorMode(0)
+        msvcrt.SetErrorMode(old)
+
+    @unittest.skipUnless(Py_DEBUG, "only avialable under debug build")
```

Comment:
```suggestion
    @unittest.skipUnless(Py_DEBUG, "only available under debug build")
```

same for the other two tests :)

Reply:
Thanks for the review!